web-console/src/views/query-view/query-output/query-output.tsx

```diff
@@ -43,13 +55,200 @@ export class QueryOutput extends React.PureComponent<QueryOutputProps> {
+              intent: Intent.SUCCESS,
+            });
+          },
+        },
+      ]);
+    } else {
+      const { sorted } = this.props;
+      const basicActions: BasicAction[] = [];
+      if (sorted) {
+        sorted.map(sorted => {
+          if (sorted.id === h) {
+            basicActions.push({
+              icon: sorted.desc ? IconNames.SORT_ASC : IconNames.SORT_DESC,
+              title: `Order by: ${h} ${sorted.desc ? 'ASC' : 'DESC'}`,
+              onAction: () => handleSQLAction('', h, 'order by'),
+            });
+          }
+        });
+      }
+      if (!basicActions.length) {
+        basicActions.push(
+          {
+            icon: IconNames.SORT_ASC,
+            title: `Order by: ${h} ASC`,
+            onAction: () => handleSQLAction('', h, 'order by'),
+          },
+          {
+            icon: IconNames.SORT_DESC,
+            title: `Order by: ${h} DESC`,
+            onAction: () => handleSQLAction('', h, 'order by'),
+          },
+        );
+      }
+      basicActions.push({
+        icon: IconNames.CROSS,
+        title: `Remove: ${h}`,
+        onAction: () => handleSQLAction('', h, 'exclude column'),
+      });
+      actionsMenu = basicActionsToMenu(basicActions);
+    }
+    return actionsMenu ? actionsMenu : undefined;
+  }
+
+  getRowActions(row: string, header: string) {
+    const { disabled, handleSQLAction } = this.props;
+    let actionsMenu;
+    if (disabled) {
+      actionsMenu = basicActionsToMenu([
+        {
+          icon: IconNames.CLIPBOARD,
+          title: `Copy '${row}'`,
+          onAction: () => {
+            copy(row, { format: 'text/plain' });
+            AppToaster.show({
+              message: `${row} copied to clipboard`,
+              intent: Intent.SUCCESS,
+            });
+          },
+        },
+        {
+          icon: IconNames.CLIPBOARD,
+          title: `Copy 'WHERE ${basicIdentifierEscape(header)} = ${basicLiteralEscape(row)}'`,
+          onAction: () => {
+            copy(`WHERE  ${basicIdentifierEscape(header)} = ${basicLiteralEscape(row)}`, {
+              format: 'text/plain',
+            });
+            AppToaster.show({
+              message: `WHERE ${basicIdentifierEscape(header)} = ${basicLiteralEscape(
+                row,
+              )} copied to clipboard`,
+              intent: Intent.SUCCESS,
+            });
+          },
+        },
+        {
+          icon: IconNames.CLIPBOARD,
+          title: `Copy 'WHERE ${basicIdentifierEscape(header)} != ${basicLiteralEscape(row)}'`,
+          onAction: () => {
+            copy(`WHERE  ${basicIdentifierEscape(header)} != ${basicLiteralEscape(row)}`, {
+              format: 'text/plain',
+            });
+            AppToaster.show({
+              message: `WHERE ${basicIdentifierEscape(header)} != ${basicLiteralEscape(
+                row,
+              )} copied to clipboard`,
+              intent: Intent.SUCCESS,
+            });
```

Comment:
I feel like we should factor out this very common `copy` + `AppToaster.show` pattern into a sweet function